volatility3/framework/plugins/windows/dlllist.py

```diff
@@ -114,7 +117,7 @@ def _generator(self, procs):
                         mod_re = re.compile(self.config["name"], flags)
                     except re.error:
                         vollog.debug(
-                            "Error parsing regular expression: %s", self.config["name"]
+                            f'Error parsing regular expression: {self.config["name"]}'
```

Comment:
Thanks, but as part of the big ruff linting we did recently all vollog messages should have explicitly been reverted *back* to format string style.  Please see https://docs.astral.sh/ruff/rules/logging-f-string/ for the more information.  If you could strip this change out, everything else looks good.  5:)